docs/source/daml/reference/index.rst

```diff
@@ -21,4 +21,3 @@ This section contains a reference to writing templates for DAML contracts. It in
    File structure <file-structure>
    Packages <packages>
    Contract keys <contract-keys>
-   Generic templates <generic-templates>
```

Comment:
Do we need to update the PDF thingy as well?

Reply:
Looks like they never made it into the PDF.